cmd/client.go

```diff
@@ -82,15 +84,24 @@ var clientCmd = &cobra.Command{
 }
 
 func promptForFilePath() (string, error) {
-	filePrompt := promptui.Prompt{
-		Default:  "test_files/example.txt",
-		Label:    "Enter File Path",
-		Validate: validateFilePath,
-	}
-
-	filepath, err := filePrompt.Run()
+	var filepath string
+	filePrompt := huh.NewForm(
+		huh.NewGroup(
+			huh.NewInput().
+				Title("Enter File Path:").
+				Prompt("? ").
+				Placeholder("test_files/example.txt").
+				Suggestions([]string{"test_files/example.txt"}).
+				// Default("test_files/example.txt").
+				Validate(validateFilePath).
+				Value(&filepath),
+		),
+	)
+
+	err := filePrompt.Run()
 	if err != nil {
-		return "test_files/example.txt", nil //Currently hardcoded the value but in production this shall be validated
+		return "", err
+		// Currently hardcoded the value but in production this shall be validated
 		// Also autocomplete is required
 	} // Maybe its better to change this to bubbletea later
```

Comment:
ye wala comment remove karde (because we changed to bubbletea)

Reply:
currently hard coded wala bhi

Reply:
done